Aplib.Core/Intent/Tactics/ITactic.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Aplib.Core.Belief;
+using Aplib.Core.Intent.Actions;
+
+namespace Aplib.Core.Intent.Tactics
+{
+    /// <summary>
+    /// Represents a tactic that an agent can use to achieve its goals.
+    /// A tactic is a strategy for achieving a particular goal.
+    /// </summary>
+    /// <typeparam name="TBeliefSet">The type of the belief set that the tactic uses.</typeparam>
+    public interface ITactic<in TBeliefSet> where TBeliefSet : IBeliefSet
+    {
+        /// <summary>
+        /// Gets the first enabled action of the tactic.
+        /// An action is a specific operation that an agent can perform.
+        /// The action returned by this method is the first action that the tactic can perform,
+        /// given the current state of the world as represented by the belief set.
+        /// </summary>
```

Comment:
Gebruik voor verdere uitleg een remark, of andere XML tags. Probeer summary een 1 liner te houden.

```suggestion
        /// <summary>
        /// Gets the first enabled action of the tactic.
        /// </summary>
        /// <remark>
        /// An action is a specific operation that an agent can perform.
        /// The action returned by this method is the first action that the tactic can perform,
        /// given the current state of the world as represented by the belief set.
        /// </remark>
```